config/software/pysnmp.rb

```diff
@@ -7,5 +7,10 @@
 
 build do
   ship_license "https://gist.githubusercontent.com/remh/519324dc1b69f7488239/raw/2bbf2888194fef8ae75651e551b61f90cb49c482/pysnmp.license"
-  command "#{install_dir}/embedded/bin/pip install -I --install-option=\"--install-scripts=#{install_dir}/bin\" #{name}==#{version}"
-end
\ No newline at end of file
+  if ohai['platform'] == 'windows'
+    pip "install -I --install-option=\"--install-scripts="\
```

Comment:
let's remove the `-I` flag here (my bad, I missed it when I removed it from the other software defs)